### PR TITLE
feat(ecs): add back plugins healthcheck

### PR DESCRIPTION
**Pull request by yakkomajuri**
Reverts PostHog/posthog#8845

### CODE REVIEW

File: production.Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ RUN apk --update --no-cache add \
     "make~=4.3" \
     "nodejs~=14" \
     "npm~=7" \
+    "curl~=7" \
```
Comment (guidoiaquinti) [inline production.Dockerfile:28]: This is not needed and it is not the image used by PostHog Cloud. See https://github.com/PostHog/posthog-cloud/blob/master/prod.web.Dockerfile

Comment (yakkomajuri) [reply production.Dockerfile:28]: hmm right - so curl is there on the cloud image

Comment (yakkomajuri) [inline production.Dockerfile:28]:
```suggestion
```

Comment (yakkomajuri) [reply production.Dockerfile:28]: found the issue: we install then uninstall curl on cloud

**Review by guidoiaquinti — APPROVED** [review]

File: production.Dockerfile (hunk 1 above)
Comment (yakkomajuri) [reply production.Dockerfile]: https://github.com/PostHog/posthog-cloud/pull/171

File: task-definition.plugins.json [hunk 2]
```diff
@@ -161,6 +161,13 @@
             ],
             "linuxParameters": {
                 "initProcessEnabled": true
+            },
+            "healthCheck": {
+                "retries": 10,
+                "command": ["CMD-SHELL", "curl -f http://localhost:6738/_ready || exit 1"],
+                "timeout": 10,
+                "interval": 45,
+                "startPeriod": 45
```
Comment (yakkomajuri) [inline task-definition.plugins.json:170]:
```suggestion
                "startPeriod": 40
```

**Pull request merged**